{
  "comments": [
    {
      "key": {
        "uuid": "ba60267b_7f297358",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 468,
      "author": {
        "id": 1000029
      },
      "writtenOn": "2021-09-21T20:54:53Z",
      "side": 1,
      "message": "What happens if the service is already running? Do we get the exception below or a different type of error message?\n\nI think we should handle the case where the `resume` is given but the service is already running by checking the service\u0027s state prior to attempting to start...providing some sort of message back to the user that fan control is already running. Reason for this is if they already ran `resume`, forgot they did, and then try and `resume` again.",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 468,
        "endChar": 48
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "0190de2e_4a3c0020",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 468,
      "author": {
        "id": 1001082
      },
      "writtenOn": "2021-09-23T19:26:28Z",
      "side": 1,
      "message": "It handles resume fine when fan-control is already running. It does emit an exception if you try to reload() while it\u0027s not running, however:\n\nroot@p10bmc:~# /tmp/build/fanctl reload\n/tmp/build/fanctl failed: DBus method failed: org.freedesktop.systemd1 /org/freedesktop/systemd1 org.freedesktop.systemd1.Manager KillUnit",
      "parentUuid": "ba60267b_7f297358",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 468,
        "endChar": 48
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "23655c6e_8f09d17e",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 468,
      "author": {
        "id": 1000029
      },
      "writtenOn": "2021-09-24T18:52:43Z",
      "side": 1,
      "message": "Ack",
      "parentUuid": "0190de2e_4a3c0020",
      "range": {
        "startLine": 465,
        "startChar": 8,
        "endLine": 468,
        "endChar": 48
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "aeda8249_5cc1aece",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 521,
      "author": {
        "id": 1000029
      },
      "writtenOn": "2021-09-21T20:54:53Z",
      "side": 1,
      "message": "It doesnt appear that this is optional:\n\n```\nroot@p10bmc:~# ./fanctl set 5000  \nRequires at least 2 options used and only 1were given from [target,fan list,-h,--help]\nRun with --help for more information.\n```",
      "range": {
        "startLine": 520,
        "startChar": 8,
        "endLine": 521,
        "endChar": 72
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "b2e62fb3_9fd5be32",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 521,
      "author": {
        "id": 1001082
      },
      "writtenOn": "2021-09-23T19:26:28Z",
      "side": 1,
      "message": "the final fix for this is in the last commit. However there was another bug that was found and fixed in the latest changeset.",
      "parentUuid": "aeda8249_5cc1aece",
      "range": {
        "startLine": 520,
        "startChar": 8,
        "endLine": 521,
        "endChar": 72
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "d4eba17a_868d11c3",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 521,
      "author": {
        "id": 1000029
      },
      "writtenOn": "2021-09-24T18:52:43Z",
      "side": 1,
      "message": "It appears there\u0027s no difference between ps38 and ps39.",
      "parentUuid": "b2e62fb3_9fd5be32",
      "range": {
        "startLine": 520,
        "startChar": 8,
        "endLine": 521,
        "endChar": 72
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "0d05b7f4_dc1bb508",
        "filename": "control/fanctl.cpp",
        "patchSetId": 38
      },
      "lineNbr": 521,
      "author": {
        "id": 1001082
      },
      "writtenOn": "2021-09-24T19:08:37Z",
      "side": 1,
      "message": "I back-ported the fix to this commit. the pathMap[\"speed\"] should be pathMap[\"tach\"],",
      "parentUuid": "d4eba17a_868d11c3",
      "range": {
        "startLine": 520,
        "startChar": 8,
        "endLine": 521,
        "endChar": 72
      },
      "revId": "63d0e5e75a59ecfe1f7099fdeacf6ebdea449e2d",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": false
    }
  ]
}